scale-tests/streaming_test.py

```diff
@@ -75,7 +75,7 @@ def _submit_consumer(broker_dns, common_conf, topic, spark_app_name, driver_role
         "--supervise",
         "--conf", "spark.mesos.containerizer=mesos",
         "--conf", "spark.mesos.driver.failoverTimeout=30",
-        "--conf", "spark.mesos.uris=http://norvig.com/big.txt",
+        "--conf", "spark.mesos.uris=https://s3-us-west-2.amazonaws.com/infinity-soak/data/norvig/big.txt",
```

Comment:
might I suggest using the bucket domain style `https://infinity-soak.s3.amazonaws.com/data/norvig/big.txt` rather than specifying region where bucket is located?

Reply:
good catch, thanks! updated